src/Libraries/RevitNodes/GeometryConversion/CurveLoopPartition.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.DesignScript.Runtime;
+using Autodesk.Revit.DB;
+using BoundingBox = Autodesk.DesignScript.Geometry.BoundingBox;
+using PolyCurve = Autodesk.DesignScript.Geometry.PolyCurve;
+
+namespace Revit.GeometryConversion
+{
+
+    [SupressImportIntoVM]
+    internal static class CurveLoopPartition
+    {
+        public static List<List<List<Curve>>> ByCurveLoopsAndFace(Face face, List<List<Curve>> curveLoops)
```

Comment:
Is the parameter `face` used in this method?

Reply:
It looks like the face is no use in this method now, I will remove it.